src/main/java/frc/robot/subsystems/Drive.java

```diff
@@ -304,8 +304,7 @@ public void periodic() {
         super.periodic();
         isBlue = DriverStation.getAlliance().orElse(Alliance.Red) == Alliance.Blue;
         estimator.update(getMap().gyro.getRotation2d(), getData().getModulePositions());
-        visionEstimator.update(getMap().gyro.getRotation2d(), getData().getModulePositions());
-        Pose3d estimatorPose3D = new Pose3d(estimator.getEstimatedPosition());
+        visionEstimator.update(getMap().gyro.getRotation2d(), getData().getModulePositions());s
```

Comment:
```suggestion
        visionEstimator.update(getMap().gyro.getRotation2d(), getData().getModulePositions());
```
Accidental typo! Don't forget to build before pushing.